package.json

```diff
@@ -153,8 +153,7 @@
     "unplugin-auto-import": "^19.1.2",
     "unplugin-vue-components": "^28.4.1",
     "vaul-vue": "^0.4.1",
-    "vue": "^3.5.13",
-    "vue-router": "^4.5.0"
+    "vue": "^3.5.13"
```

Comment:
I guess you can remove `vue` as well? I've added it at the same time as `vue-router` to partially fix the issue: https://github.com/nuxt/ui/commit/c713f3015de7f1fad0dd37e881c8209b04c66f28